### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by vcolavin**
@orangewolf

### CODE REVIEW

File: spec/features/signup_spec.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  after :all do
+    Course.destroy_all
+  end
+
+
+  context 'full time' do
+    # this should work, and very occasionally does, but usually doesn't.
+    it "submits the application with valid data", js: true do
+      visit "/apply"
```
Comment (orangewolf) [inline spec/features/signup_spec.rb]: this doesn't seem like the right location to me

Comment (vcolavin) [inline spec/features/signup_spec.rb]: It'll work. We'd have to click the "apply" link, which takes us to the "/apply" route. That didn't seem like a part of the application process to me, but I can add it very easily.

File: spec/features/signup_spec.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      click_on("Jumpstart")
+      page.find('#uiSelect').click
+
+      page.find('#ui-select-choices-1').click
+
+      # capybara needs help clicking the button, so scroll down
+      page.execute_script('window.scrollBy(0,400)')
+      page.find('button.btn.enrollment').click
+
+      expect(page).to have_text("???")
```
Comment (winescout) [inline spec/features/signup_spec.rb]: Is this a WIP?

Comment (vcolavin) [inline spec/features/signup_spec.rb]: Yes. Rob asked us to put up a PR so he could review.

File: spec/features/signup_spec.rb (hunk 1 above)
Comment (orangewolf) [inline spec/features/signup_spec.rb]: @vcolavin there is an application form ON the home page, the point is to try that application form, not just jump off from there to the other page.

**Pull request merged**